### PR TITLE
update japanese doc

### PR DESCRIPTION
**Pull request by kotarella1110**

### CODE REVIEW

**Comment by bluebill1049** [comment]
hey @kotarella1110 you want to update all the links as well? `/jp` we going live soon anyhow :)

File: README.ja-JP.md [hunk 1]
```diff
@@ -54,7 +54,7 @@
 - [例](https://github.com/bluebill1049/react-hook-form/tree/master/examples)
 - [デモ](https://react-hook-form.com)
 - [Form Builder](https://react-hook-form.com/form-builder)
-- [FAQ](https://react-hook-form.com/faq)
+- [FAQs](https://react-hook-form.com/faq)
```
Comment (bluebill1049) [inline README.ja-JP.md]: let's go!

https://react-hook-form.com/jp

**Comment by kotarella1110** [comment]
@bluebill1049 That's a good idea!
I update all the links!

**Comment by kotarella1110** [comment]
completed!

**Pull request merged**